screens/ResourceScreen.js

```diff
@@ -25,17 +26,9 @@ const ResourceScreen = () => {
 
 const styles = StyleSheet.create({
   container: {
```

Comment:
You can rename the styles object. Be sure to change the name inside the <Button> components too
```suggestion
  button: {
```